scania-digital-design-system/sdds
# Improvement/side menu in React demo

**Describe pull-request**  
Added side menu example instead of header menu in React demo

**Solving issue**  
Fixes: #AB1154

**How to test**  
_Add description how to test if possible_
1. Run React demo 
2. Check in side menu 
     - Dropdown menu items should open sub-menu items 
     - Responsive(Mobile) view: The menu is always present on the left side of the header in these viewports.
     - Toggle between the expanded and collapsed state by pressing the expand/collapse link in the bottom of the side menu and the pop-over appears at hover of sub-menu item

**Known issue**
- Focus state(outline) on menu items don't cover the whole item box area. It is related to shared classes with header
- Active status example is missing
